### PR TITLE
Added HTML and SCSS To Leadership team section

### PR DESCRIPTION
**Pull request by golaughlin**
Added the HTML and SCSS needed for the Leadership Team section.

### CODE REVIEW

**Review by daleinen7 — CHANGES_REQUESTED** [review]
* Looks like the header section above the cards doesn't have a set width. Here's the specs for the "Work With Us" section that you can copy from, but it's in the figma. The size changes as the window shrinks. 
You may be able to refactor the stylings of that section to work with this section too. 

Screen Shot 2021-07-01 at 8 19 19 PM

* It also looks like the spacing of the cards is off as well. 
The specs have the second row evenly spaced with the above row.
Screen Shot 2021-07-01 at 8 27 25 PM

As is, there is a gap on the bottom row, and the gaps on the top row is not consistent.
Screen Shot 2021-07-01 at 8 32 49 PM

* Also when they go into two rows they look like they don't have a consistent spacing like a lot of the other cards elsewhere. 
Screen Shot 2021-07-01 at 8 30 27 PM

File: pages/about.html [hunk 1]
```diff
@@ -4,6 +4,48 @@
...
+        </div>
+        <div class="leader-card">
+            <img alt="Jamal Riley" src="\images\proxy-img.png">
+            <!--Replace proxy image when actual picture becomes avaiable-->
+            <h3>Jamal Riley</h3>
+            <p>Volunteers Co-Lead</p>
+        </div>
+   </div>
+</section>
+=======
```
Comment (jrapala) [inline pages/about.html:48]: Please remove this conflict marker

File: scss/main.scss [hunk 2]
```diff
@@ -1965,6 +1965,26 @@ p.cardText {
...
+    color: $black;
+  }
+}
+
+.leader-cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+=======
```
Comment (jrapala) [inline scss/main.scss:1987]: Please remove this conflict marker

**Review by jrapala — CHANGES_REQUESTED** [review]

**Comment by Yeen125** [comment]
I have fixed the SCSS for the Header portion. As for the cards, the spacing is an issue with Flexbox. So if you have any suggestions or code on how to fix that, please let me know ASAP.

File: Gemfile.lock [hunk 3]
```diff
@@ -126,4 +126,4 @@ DEPENDENCIES
   webrick
 
 BUNDLED WITH
-   2.1.4
+   2.2.9
```
Comment (daleinen7) [inline Gemfile.lock:129]: I'm not sure what this means. Maybe @jrapala knows? I'm guessing this is fine, but I just don't know enough about Ruby to say.

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Closer. There are still some sizes where the header doesn't match the figma. 

Also I noticed that this section is at the top, and I believe it's supposed to go below the "Why Join Us" section. 

As far as the cards, I found this [Stack Overflow Answer](https://stackoverflow.com/questions/46276793/how-to-center-elements-on-the-last-row-in-css-grid) that should point you in the right direction. As far as centering them, it's as easy as `justify-content: center;` but the trick is how do you keep the gap between each of them consistent. 

Might have to do a bit of research for that (if you've never read it [this article](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is great at explaining what Flex can do). Let me know if you need any further help with it.

File: Gemfile.lock (hunk 3 above)
Comment (jrapala) [reply Gemfile.lock:129]: Good looking out. It's just the version of Bundler being used. We can leave this.

File: scss/main.scss [hunk 4]
```diff
@@ -1990,14 +1990,24 @@ p.cardText {
...
   display: flex;
   flex-wrap: wrap;
-  justify-content: space-evenly;
+  justify-content: center;
+  margin: auto;
+  max-width: 1440px; 
 }
 
+.leader-card {
+  margin: 10px;
```
Comment (daleinen7) [inline scss/main.scss:2008]: Looks like the margin is a little bigger in the figma

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Closer! Just two more changes and I think it will be good. 

Screen Shot 2021-07-15 at 4 11 53 PM

The figma has a 50px gutter between the cards. Otherwise the section is looking good to me. It still is at the top of the page, and I think it's supposed to be at the bottom just above the footer.

File: scss/main.scss [hunk 5]
```diff
@@ -1965,6 +1965,49 @@ p.cardText {
...
+      margin-bottom: 24px;
+    }
+  }
+
+  h2, h3, p {
+    color: $black;
+  }
+
+  * {
+    box-sizing: border-box;
```
Comment (jrapala) [inline scss/main.scss:1995]: Can you please remove? We already do this in the CSS reset.

**Review by jrapala — CHANGES_REQUESTED** [review]

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Looks like this section is still at the top. It should be at the bottom. Here is the figma for reference:
Screen Shot 2021-07-17 at 7 13 29 PM

**Comment by Yeen125** [comment]
Just moved the Leadership Team section to the bottom of the page from the top. If it's not visible, it's because I'm using my laptop instead of the PC I have been using originally for the pull request.

**Review by jrapala — CHANGES_REQUESTED** [review]
Can you please match the designs? Here are two examples of inconsistencies:

**PR:**
Screen Shot 2021-08-02 at 5 16 47 PM

**Design:**
Screen Shot 2021-08-02 at 5 17 04 PM

**PR:**
Screen Shot 2021-08-02 at 5 17 12 PM

**Design:**
Screen Shot 2021-08-02 at 5 17 20 PM

**Comment by Ryan-Koch** [comment]
I made a couple of minor changes that might address what was being asked for here. Admittedly though the comment thread on this is long so it's entirely possible I missed something. It looked like the main thing left was around the way the leadership cards were setup and the width of the bg color for the 'who is' section. The design also had a previous member of the leadership team listed, so I went ahead and removed that card as well.

Do these changes get us to a good spot with this or are there other things needed?

**Review by daleinen7 — CHANGES_REQUESTED** [review]
Just one padding issue I noticed. Otherwise, it looked good to me.

**Pull request closed without merging**